controllers/numaresourcesscheduler_controller.go

```diff
@@ -153,6 +156,22 @@ func (r *NUMAResourcesSchedulerReconciler) reconcileResource(ctx context.Context
 
 }
 
+func (r *NUMAResourcesSchedulerReconciler) getRelatedObjects(dpStatus nropv1.NamespacedName) []configv1.ObjectReference {
+	// 'Resource' should be in lowercase and plural
```

Comment:
could we have a quote (link to doc, source code) for this?

Reply:
https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names

> Most resource types require a name that can be used as a DNS subdomain name as defined in [RFC 1123](https://tools.ietf.org/html/rfc1123). This means the name must:
> 
> - contain no more than 253 characters
> - contain only lowercase alphanumeric characters, '-' or '.'
> - start with an alphanumeric character
> - end with an alphanumeric character

Reply:
sure, let's add this info to the comment. Thanks!